chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -684,18 +526,267 @@ def post_transition(self):
+    text_template = u'inforequests/obligee_action/texts/was_accepted.html'
+
+    def add_fields(self):
+        super(WasItAccepted, self).add_fields()
+
+        self.fields[u'was_accepted'] = forms.ChoiceField(
+                label=u' ',
+                choices=(
+                    (u'all', _(u'inforequests:obligee_action:WasItAccepted:all')),
+                    (u'some', _(u'inforequests:obligee_action:WasItAccepted:some')),
+                    (u'none', _(u'inforequests:obligee_action:WasItAccepted:none')),
+                    ),
+                widget=forms.RadioSelect(),
+                )
+
+    def post_transition(self):
+        res = super(WasItAccepted, self).post_transition()
+
+        if not self.is_valid():
+            res.next = WasItReturned
+        elif self.cleaned_data[u'was_accepted'] == u'none':
+            res.next = CanAddAffirmation
+        else:
+            res.next = WasItReturned
+
+        return res
+
+class ContainsAppealInfo(ObligeeActionStep):
+    label = _(u'inforequests:obligee_action:ContainsAppealInfo:label')
+    text_template = u'inforequests/obligee_action/texts/contains_appeal_info.html'
+    global_fields = [u'disclosure_level']
+    post_step_class = WasItAccepted
+
+    def add_fields(self):
+        super(ContainsAppealInfo, self).add_fields()
+
+        self.fields[u'disclosure_level'] = forms.TypedChoiceField(
+                label=u' ',
+                coerce=int,
+                choices=(
+                    (Action.DISCLOSURE_LEVELS.FULL,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:full')),
+                    (Action.DISCLOSURE_LEVELS.PARTIAL,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:partial')),
+                    (Action.DISCLOSURE_LEVELS.NONE,
+                        _(u'inforequests:obligee_action:ContainsAppealInfo:none')),
+                    ),
+                widget=forms.RadioSelect(),
+                )
+
+class IsItAppealDecision(ObligeeActionStep):
+    label = _(u'inforequests:obligee_action:IsItAppealDecision:label')
+    text_template = u'inforequests/obligee_action/texts/is_appeal_decision.html'
+
+    def add_fields(self):
+        super(IsItAppealDecision, self).add_fields()
+
+        self.fields[u'is_decision'] = forms.TypedChoiceField(
+                label=u' ',
+                coerce=int,
+                choices=(
+                    (1, _(u'inforequests:obligee_action:IsItAppealDecision:yes')),
+                    (0, _(u'inforequests:obligee_action:IsItAppealDecision:no')),
+                    ),
+                widget=forms.RadioSelect(),
+                )
+
+    def post_transition(self):
+        res = super(IsItAppealDecision, self).post_transition()
+
+        if not self.is_valid():
+            branch = self.wizard.values.get(u'branch', None)
+            if branch.can_add_disclosure_refusal_advancement_extension:
```

Comment:
Na druhe zamyslenie, imho kaslime na riesenie, ktorou vetvou wizard dalej pustit, ak tento formular nie je validne vyplneny. A pustime ho v takom pripade vzdy do kroku `ContainsAppealInfo`, ako keby bola odpoved "yes". Kedze toto rozhodovanie, ze ci `CanAddDisclosureRefusalAdvancementExtension` alebo `ContainsAppealInfo` nie je uplne zjavne, preco to robime.

Jediny zly nasledok to bude mat, ze odhadovany pocet krokov vyzardu bude obcas zly.